docs/sidebars.js

```diff
@@ -87,31 +87,30 @@ module.exports = {
     {
       type: 'category',
       label: 'Introduction',
-      items: ['distributors/introduction/overview'],
+      items: ['distributors/introduction/overview', 'distributors/introduction/onboarding'],
       collapsed: false,
     },
     {
       type: 'category',
       label: 'Specification',
+      items: ['distributors/specification/overview', 'distributors/specification/metadata'],
+      collapsed: false,
+    },
+    {
+      type: 'category',
+      label: 'Deal Types',
       items: [
-        'distributors/specification/overview',
-        'distributors/specification/metadata',
-        {
-          type: 'category',
-          label: 'Deal Types',
-          items: [
-            'distributors/specification/deal-types/recommended',
-            'distributors/specification/deal-types/supported-deal-types',
-          ],
-          collapsed: false,
-        },
+        'distributors/specification/deal-types/supported-deal-types',
```

Comment:
Can we keep the recommended page first? I authored that page b/c some folks were confused with where to start, given that our docs spell out everything that's possible, but the recommended thing is more of the "quick how-to"

If we want, we could call it the "Overview" page for deal types and combine with Supported Deal Types